s/api/types/auth.ts

```diff
@@ -1,33 +1,34 @@
-
-import * as dbmage from "dbmage"
-import {AppDatabase} from "./schema.js"
+import * as dbmage from "dbmage";
+import { AppDatabase } from "./schema.js";
```

Comment:
whitespace -- mind the consistency. we don't use spaces around imports or within object literals.